app/containers/App/Sidebar/Sidebar.jsx

```diff
@@ -53,6 +54,11 @@ const Sidebar = ({ className, theme }: Props) => {
           className={styles.navItem}
           activeClassName={styles.active}
         >
+          {!!pendingTransactionsCount && (
```

Comment:
I actually think `pendingTransactionsCount > 0` reads a bit better.. but perhaps this just a personal preference.